src/main/java/com/artipie/auth/CachedAuth.java

```diff
@@ -18,53 +20,113 @@
  * instead of calling origin authentication.
  * </p>
  * @since 0.10
- * @todo #285:30min Specify expiration time configuration.
- *  Instead of using scheduled executor to clean-up all cache map, use
- *  some configuration to clean-up only expired items, e.g. if token was not accessed for
- *  X minutes, then remove only this token. Consider using Guava's time-evicted-cache
- *  implementation: https://github.com/google/guava/wiki/CachesExplained#eviction
- * @todo #442:30min Refactor this class to move cache field to instance field
- *  and make sure the instance is not created on each request. Right now we're using
- *  a hotfix - CachedAuth created from settings on each request but uses static hash map
- *  field for caching.
  */
-public final class CachedAuth implements Authentication {
-
+public final class CachedAuth implements AuthCache {
```

Comment:
I'd suggest to think about other name for this class: for example, we can name it by expiration method `ExpiredAfterAccess` or by cache implementation used inside.

Reply:
@olenagerasimova I don't think that `ExpiredAfterAccess` is more suitable because it is possible that internal realization will change. For example `expireAfterWrite` or something else. Renamed to `CachedUsers`. Hope it will be a bit clearer